iwyu_ast_util.cc

```diff
@@ -942,12 +942,21 @@ bool IsFriendDecl(const Decl* decl) {
   return decl->getFriendObjectKind() != Decl::FOK_None;
 }
 
+bool IsExplicitInstantiation(const clang::CXXRecordDecl* decl) {
+  const auto kind = decl->getTemplateSpecializationKind();
```

Comment:
The name of the getter contains the enum type, *TemplateSpecializationKind*. That's why I thought it would be clear. I can repeat myself, no problem ;)